src/Universe.cpp

```diff
@@ -709,22 +714,33 @@ void Universe::calculateBoundaries() {
       halfspace = s_iter->second->_halfspace;
 
       if (surf->getSurfaceType() == XPLANE && halfspace == +1 &&
-          surf->getBoundaryType() != BOUNDARY_NONE)
-        min_x = surf->getMinX(halfspace);
+          surf->getBoundaryType() != BOUNDARY_NONE) {
+        if(surf->getMinX(halfspace) > min_x) {
+          min_x = surf->getMinX(halfspace);
```

Comment:
I think there may be a problem there. 
If there is three cells ||, two on top, one below, inside a universe {} :
 {| 1  | 2 |}
 ----{| 3 |}
Then the min_x of the universe is on the left cell. But when checking the bottom cell's surfaces, we'll find a bigger min_x.

So the inequality should be flipped, and special care should be taken with that initial infinite value